backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ProjectDatabaseHandler.java

```diff
@@ -404,6 +406,51 @@ public RequestStatus updateProject(Project project, User user) throws SW360Excep
         }
     }
 
+    private void setRequestedDateAndTrimComment(Project project, Project actual, User user) {
+        Set<String> actualReleaseIds = null;
+        if (Objects.nonNull(actual)&&Objects.nonNull(actual.getReleaseIdToUsage())) {
+            actualReleaseIds = CommonUtils.nullToEmptySet(actual.getReleaseIdToUsage().keySet());
+        }
+        final Set<String> actualReleaseIdsFinal = CommonUtils.nullToEmptySet(actualReleaseIds);
+        Set<String> updatedReleaseIds = null;
+        if (Objects.nonNull(project.getReleaseIdToUsage())) {
+            updatedReleaseIds = CommonUtils.nullToEmptySet(project.getReleaseIdToUsage().keySet());
+        } else {
+            updatedReleaseIds = new HashSet<>();
+        }
+
+        updatedReleaseIds.stream().filter(updatedReleaseId -> !actualReleaseIdsFinal.contains(updatedReleaseId))
+                .forEach(updatedReleaseId -> {
+                    ProjectReleaseRelationship projectReleaseRelationship = project.getReleaseIdToUsage()
+                            .get(updatedReleaseId);
+                    if (Objects.nonNull(projectReleaseRelationship))
+                        projectReleaseRelationship.setCreatedOn(SW360Utils.getCreatedOn());
+                    projectReleaseRelationship.setCreatedBy(user.getEmail());
```

Comment:
please use `braces` as there are more than one statement inside `if` clause.
Same applies to `if` clause in line `437`.